Celeste.Mod.mm/Patches/Level.cs

```diff
@@ -206,13 +206,15 @@ class patch_Level : Level {
                     Pause();
                 }
             } catch (Exception e) {
-                if (e is ArgumentOutOfRangeException && e.StackTrace.Contains("get_DefaultSpawnPoint")) {
-                    patch_LevelEnter.ErrorMessage = Dialog.Get("postcard_levelnospawn");
-                } else {
-                    patch_LevelEnter.ErrorMessage = Dialog.Get("postcard_levelloadfailed").Replace("((sid))", Session.Area.GetSID());
+                if (patch_LevelEnter.ErrorMessage == null) {
+                    if (e is ArgumentOutOfRangeException && e.StackTrace.Contains("get_DefaultSpawnPoint")) {
```

Comment:
I complained about all the other `StackTrace.Contains`, so I might as well mention this one, though I'm not sure how best to address it. If we wanted to get rid of it, that would probably involve adding a bounds check in the `DefaultSpawnPoint` getter.